AnkiConnect.py

```diff
@@ -417,6 +417,22 @@ def deckNames(self):
             return collection.decks.allNames()
 
 
+    def launchBrowser(self, query):
+        browser = aqt.dialogs.open("Browser", self.window())
+        browser.activateWindow()
+        if query:
+            query = u'"{}"'.format(query)
+            browser.form.searchEdit.lineEdit().setText(query)
+            browser.onSearch()
+        return browser.model.cards
+
+
+    def launchAddCards(self):
+        addcards = aqt.dialogs.open("AddCards", self.window())
+        addcards.activateWindow()
+        return 0
```

Comment:
Prefer to return `None` instead of an arbitrary value of `0`. This can be done implicitly by just deleting `return 0` (don't have to add `return None`.